inc/gui.class.php

```diff
@@ -160,74 +160,74 @@ public static function options_page() { ?>
 								<label for="ab_gravatar_check">
-									<?php esc_html_e('Trust commenters with a Gravatar', 'antispam-bee') ?>
-									<span><?php _e('Check if commenter has a Gravatar image', 'antispam-bee') ?></span>
+									<?php esc_html_e( 'Trust commenters with a Gravatar', 'antispam-bee' ); ?>
+									<span><?php esc_html_e( 'Check if commenter has a Gravatar image', 'antispam-bee' ); ?></span>
 								</label>
 							</li>
 
 							<li>
 								<input type="checkbox" name="ab_time_check" id="ab_time_check" value="1" <?php checked($options['time_check'], 1) ?> />
 								<label for="ab_time_check">
-									<?php esc_html_e('Consider the comment time', 'antispam-bee') ?>
-									<span><?php esc_html_e('Not recommended when using page caching', 'antispam-bee') ?></span>
+									<?php esc_html_e( 'Consider the comment time', 'antispam-bee' ); ?>
+									<span><?php esc_html_e( 'Not recommended when using page caching', 'antispam-bee' ); ?></span>
 								</label>
 							</li>
 
 							<li>
 								<input type="checkbox" name="ab_bbcode_check" id="ab_bbcode_check" value="1" <?php checked($options['bbcode_check'], 1) ?> />
 								<label for="ab_bbcode_check">
-									<?php esc_html_e('BBCode is spam', 'antispam-bee') ?>
-									<span><?php esc_html_e('Review the comment contents for BBCode links', 'antispam-bee') ?></span>
+									<?php esc_html_e( 'BBCode is spam', 'antispam-bee' ); ?>
+									<span><?php esc_html_e( 'Review the comment contents for BBCode links', 'antispam-bee' ); ?></span>
 								</label>
 							</li>
 
 							<li>
 								<input type="checkbox" name="ab_advanced_check" id="ab_advanced_check" value="1" <?php checked($options['advanced_check'], 1) ?> />
 								<label for="ab_advanced_check">
-									<?php esc_html_e('Validate the ip address of commenters', 'antispam-bee') ?>
-									<span><?php esc_html_e('Validity check for used ip address', 'antispam-bee') ?></span>
+									<?php esc_html_e( 'Validate the ip address of commenters', 'antispam-bee' ); ?>
+									<span><?php esc_html_e( 'Validity check for used ip address', 'antispam-bee' ); ?></span>
 								</label>
 							</li>
 
 							<li>
 								<input type="checkbox" name="ab_regexp_check" id="ab_regexp_check" value="1" <?php checked($options['regexp_check'], 1) ?> />
 								<label for="ab_regexp_check">
-									<?php esc_html_e('Use regular expressions', 'antispam-bee') ?>
-									<span><?php _e('Predefined and custom patterns by <a href="https://gist.github.com/4242142" target="_blank">plugin hook</a>', 'antispam-bee') ?></span>
+									<?php esc_html_e( 'Use regular expressions', 'antispam-bee' ); ?>
+									<span><?php esc_html_e( 'Predefined and custom patterns by <a href="https://gist.github.com/4242142" target="_blank" rel="noopener noreferrer">plugin hook</a>', 'antispam-bee' ); ?></span>
```

Comment:
And as the translation contains HTML I think we can't escape it, because it would show the escaped link instead of link then.